unlock-app/src/components/creator/lock/LockIconBar.tsx

```diff
@@ -28,27 +38,33 @@ export function LockIconBar({ lock, toggleCode, withdraw, toggleCreditCard }) {
     )
   }
   const membersPage = `/members?locks=${lock.address}`
+  const verifiersPage = `/verifiers?lockAddress=${lock.address}`
+  const { explorer } = config.networks[network!] ?? {}
   // Otherwise, we just show the lock icon bar
   return (
-    <StatusBlock>
-      <IconBar>
-        <Buttons.CreditCard as="button" lock={lock} action={toggleCreditCard} />
-        <Buttons.Withdraw as="button" lock={lock} withdraw={withdraw} />
-        <Buttons.Demo
-          as="a"
-          href={`/demo?network=${network}&lock=${lock.address}`}
+    <div
+      className="flex flex-wrap justify-around"
+      style={{
+        maxWidth: '250px',
+      }}
```

Comment:
Nit, you can do the same in className using `max-w-[250px]`. I would use a different measurement unit than pixels though.

Reply:
oh cool! thk!